lib/chitter_manager.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class ChitterManager
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps ORDER BY peep_date DESC;")
+    result.map do |peep| 
+      ChitterManager.new(id: peep['id'], message: peep['message'], date: peep['peep_date'])
+    end
+  end
+
+  def self.post(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    peep = message.gsub(/'/, "''")
```

Comment:
Interesting. What caused you to add this?

Reply:
When I was testing I realised that peeps such as "It's Saturday" or "They're here" would escape my database insertion command and throw errors. I was familiar with regex from some javascript tutorials I had done last year so I looked for a Ruby solution and found gsub to insert another single quote and stop it escaping when going into the database.